internal/service/advancedclustertpf/resource_unit_test.go

```diff
@@ -0,0 +1,332 @@
+package advancedclustertpf_test
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+
+	"github.com/mongodb/terraform-provider-mongodbatlas/internal/service/advancedclustertpf"
+	"github.com/mongodb/terraform-provider-mongodbatlas/internal/testutil/acc"
+	"github.com/mongodb/terraform-provider-mongodbatlas/internal/testutil/unit"
+)
+
+const (
+	resourceName        = "mongodbatlas_advanced_cluster.test"
+	processResponseOnly = "processResponseOnly"
+	projectID           = "111111111111111111111111"
+	clusterName         = "test"
+)
+
+func TestMockAdvancedCluster_replicaset(t *testing.T) {
+	var (
```

Comment:
nit: const instead of var

Reply:
prefer keeping it as is instead of change between `var` and `const` but happy to change if you insist :D